sigs/addons/MIGRATION_TO_CORE.md

```diff
@@ -0,0 +1,32 @@
+# Migration From TF-Addons To TensorFlow Core
+
+### In-Progress & Previous Migrations:
+https://github.com/tensorflow/addons/projects/2/
+
+### Process 
+1. Create an issue in TensorFlow Addons for a candidate that you think should be 
+migrated. 
+2. The SIG will evaluate the request and add it to the `Potential Candidates` section 
+of our GitHub project.
+3. If it's agreed that a migration makes sense, an RFC needs to be written to discuss 
+the move with a larger community audience. 
+    * If the transition will impact tf-core and Keras then submit the RFC to 
+    [TensorFlow Community](https://github.com/tensorflow/community)
+    * Additions which only subclass Keras APIs should submit their migration proposals to 
+    [Keras Governance](https://github.com/keras-team/governance)
+    
+4. A sponsor from the TF/Keras team must agree to shepard the transition.
```

Comment:
shepherd the transition and take over maintenance of the new API

Reply:
Updated.